src/discordas.ts

```diff
@@ -55,8 +68,19 @@ function generateRegistration(reg, callback)  {
     callback(reg);
 }
 
-// tslint:disable-next-line no-any
-type callbackFn = (...args: any[]) => Promise<any>;
+interface IBridgeCallbacks {
+    onAliasQueried?: (alias: string, roomId: string) => Promise<void>;
```

Comment:
why have this and the next one as optional if they are set always anyways? That results in unneeded usage of `!` all over

Reply:
No need for that, indeed. I mixed it up with the actual callbacks of the bridge, which are optional.